buildsystem/scripts/EmbedWinDependencies.cmake

```diff
@@ -37,19 +52,47 @@ function(resolve binary)
 	endforeach()
 endfunction()
 
-foreach(file ${CMAKE_INSTALL_MANIFEST_FILES})
-	if(file MATCHES "\\.dll$")
-		resolve("${file}")
-		if(windeployqt)
-			execute_process(COMMAND "${CMAKE_COMMAND}" -E env "PATH=${vcpkg_dir}/bin;$ENV{PATH}"
-				"${windeployqt}"
-				--qmldir "${CMAKE_INSTALL_PREFIX}/${asset_dir}/qml"
-				--dir "${CMAKE_INSTALL_PREFIX}/${py_install_prefix}"
-				"${file}"
-			)
+
+# windeployqt
+if(use_windeployqt AND windeployqt)
+	foreach(file ${CMAKE_INSTALL_MANIFEST_FILES})
+		if(file MATCHES "\\.dll$")
+			resolve("${file}")
+			if(windeployqt)
+				execute_process(COMMAND "${CMAKE_COMMAND}" -E env "PATH=${vcpkg_dir}/bin;$ENV{PATH}"
+					"${windeployqt}"
+					--qmldir "${CMAKE_INSTALL_PREFIX}/${asset_dir}/qml"
+					--dir "${CMAKE_INSTALL_PREFIX}/${py_install_prefix}"
+					"${file}"
+				)
+			endif()
 		endif()
-	endif()
-endforeach()
+	endforeach()
+endif()
+
+# applocal
+if((DEFINED POWERSHELL_COMMAND) AND (DEFINED APPLOCAL_SCRIPT))
+	foreach(file ${CMAKE_INSTALL_MANIFEST_FILES})
+		if(file MATCHES "\\.dll$")
+				# DEBUG
+				# message(STATUS "Use this command manually for diagnosing: ${POWERSHELL_COMMAND} ${APPLOCAL_SCRIPT} -targetBinary ${file} -installedDir ${vcpkg_dir}/bin -tlogFile ${BUILDSYSTEM_DIR}/applocal.log -copiedFilesLog ${BUILDSYSTEM_DIR}/applocal_copied_files.log!")
+
+				execute_process(COMMAND "${POWERSHELL_COMMAND}" "${APPLOCAL_SCRIPT}"
+						-targetBinary "${file}"
+						-installedDir "${vcpkg_dir}/bin"
+						-tlogFile "${BUILDSYSTEM_DIR}/applocal.log"
```

Comment:
We should avoid writing to the source directory, even for packaging (`BUILDSYSTEM_DIR` being a subdirectory).
Maybe we can add a designated directory for the packaging logs, or write to the binary directory.

Same for the few lines below.

Reply:
done